docs/_assets/codepen.js

```diff
@@ -102,6 +102,15 @@ CodeBlockCodePen.prototype.clickHandler = function(form, pre) {
 
   return function() {
 
+    // Track codepen button clicks
+    if (typeof ga !== 'undefined') {
+      ga('send', {
+        hitType: 'event',
+        eventCategory: 'codepen',
+        eventAction: 'click'
```

Comment:
You should definitely add a `eventLabel` so we know *which* codepen button has been clicked.